.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+---
+services:
+  devcontainer:
+    build:
+      context: ../  # Start the context at the project root
+      dockerfile: .devcontainer/Dockerfile
+    volumes:
```

Comment:
Should we add a comment here that a user could mount their own code (from their host system) into a new volume? Then they could run Statick using Ubuntu 24.04-based tools on their code. If so, it would be good to test that use case. And, we should install more VS Code extensions for the languages we support with discovery. I'm fine if all of that is a follow-on issue where we can have more discussion and testing.

Reply:
After talking more offline, we decided we want this dev container to be for developing Statick. The rest of this is outside the scope of what we want to maintain. It would be possible for other developers to use this dev container or our main Docker image to do their own development, but that will be up to them.